cdap-ui/app/styles/themes/cdap/header.less

```diff
@@ -1,145 +1,226 @@
 @import "../../variables.less";
 @import "../../../../bower_components/bootstrap/less/mixins.less";
 
-body.theme-cdap header {
-  background-color: @cdap-header;
-
-  [my-navbar] {
-
-    padding-top: 10px;
-    padding-bottom: 15px;
-
-    .navbar-brand {
-      position: relative;
-      top: -5px;
-
-      > strong {
-        display: inline-block;
-        visibility: hidden;
-        width: 84px;
+body {
+  header [my-navbar] {
```

Comment:
So now if I change the theme would I still be getting navbar just the same? We have header.less inside /styles/themes/cdap/header.less but we are not namespace'ing it with the theme.